Library/Homebrew/cmd/untap.rb

```diff
@@ -1,12 +1,11 @@
-# typed: strict
+# typed: true
```

Comment:
Would be good to fix up anything that requires downgrading here.

Reply:
+1, should need to `T.let` the new ivars

Reply:
Okay, I updated that.